app/helpers/fuzzy_time.rb

```diff
@@ -24,7 +24,7 @@ def ago(timestamp)
                      end
           "<span data-toggle='tooltip' title='#{timestamp.strftime('%e %B %Y at %H:%M %Z')}'>#{response}</span>"
```

Comment:
Yeah, I think that would be great.

This is used in the same place as the other one—it just depends on how long ago it was.